.github/workflows/fedora.yml

```diff
@@ -0,0 +1,224 @@
+name: Fedora (RHEL), Python 3.9
```

Comment:
Is Python3.9 already installed in the `fedora:33` image?

Reply:
python is installed by install dependencies script https://github.com/openvinotoolkit/openvino/blob/master/install_build_dependencies.sh#L129